docs/api/api.md

```diff
@@ -135,42 +135,36 @@ type CertificateRequestPolicyAllowed struct {
     CommonName *CertificateRequestPolicyAllowedString `json:"commonName,omitempty"`
 
     // DNSNames defines the X.509 DNS SANs that may be requested for.
-    // Accepts wildcards "*".
     // +optional
     DNSNames *CertificateRequestPolicyAllowedStringSlice `json:"dnsNames,omitempty"`
 
-    // IPAddresses defines the X.509 IP SANs that may be requested
-    // for.
+    // IPAddresses defines the X.509 IP SANs that may be requested for.
     // +optional
     IPAddresses *CertificateRequestPolicyAllowedStringSlice `json:"ipAddresses,omitempty"`
 
     // URIs defines the X.509 URI SANs that may be requested for.
     // +optional
     URIs *CertificateRequestPolicyAllowedStringSlice `json:"uris,omitempty"`
 
-    // EmailAddresses defines the X.509 Email SANs that may be
-    // requested for.
+    // EmailAddresses defines the X.509 Email SANs that may be requested for.
     // +optional
     EmailAddresses *CertificateRequestPolicyAllowedStringSlice `json:"emailAddresses,omitempty"`
 
     // IsCA defines whether it is permissible for a CertificateRequest to have
-    // the `spec.IsCA` field set to `true`.
-    // An omitted field, value of `nil` or `false`, forbids the `spec.IsCA` field
-    // from bring `true`.
-    // A value of `true` permits CertificateRequests setting the `spec.IsCA` field
-    // to `true`.
+    // the `spec.isCA` field set to `true`.
+    // Only a value of `true` permits CertificateRequests setting the `spec.isCA`
+    // field to `true`.
     // +optional
     IsCA *bool `json:"isCA,omitempty"`
 
     // Usages defines the list of permissible key usages that may appear
     // on the CertificateRequest `spec.keyUsages` field.
-    // An omitted field or value of `nil` forbids any Usages being requested.
-    // An empty slice `[]` is equivalent to `nil`.
+    // An omitted field or empty list forbids any key usages being requested.
```

Comment:
Could we rephrase this sentence so it is even easier to understand (eg. https://github.com/cert-manager/cert-manager/blob/master/pkg/apis/certmanager/v1/types_certificate.go#L252-L254)?

Reply:
@inteon I am unsure if I can improve the text with the link provided. In this case, it's a pointer to a slice of `KeyUsage` (alias for `string`), which is a bit different. I would prefer to avoid the term `nil`, which is a Go concept - and does not make sense IMO to use in Kubernetes API docs for yaml/json documents.

Reply:
I think this should be ok now. Please add suggestions to the source file: types_certificaterequestpolicy.go. 😉